ci/run_ci.sh

```diff
@@ -56,7 +56,7 @@ wait_clean
 for dir in ${test_list}; do
   start_time=`date`
   figlet "CI test for ${dir}"
-  if $dir/ci_test.sh; then
+  if src/$dir/ci_test.sh; then
```

Comment:
This need to be updated to:
```suggestion
   if $dir/ci_test.sh; then 
```
And -maxdepth needs to set to 2 in the previous `find` statement

Reply:
@rsevilla87  This calls the individual tests in the benchamark directories, which are located in the src/ 
I guess, this needs not to be changed.

Reply:
Yeah, but ${dir} already contains the directory structure where ci_test.sh is, which it's populated by :

test_list=`find * -maxdepth 2 -name ci_test.sh -type f -exec dirname {} \;`    <-- Note maxdepth 2
and
test_list=`echo "${diff_list}" | awk -F "/" '{print $1"/"$2}' | uniq`            <-- Note the modified print statement